release-notes/8.0/8.0.1/api-diff/Microsoft.NETCore.App/8.0.1_System.Runtime.InteropServices.JavaScript.md

```diff
@@ -0,0 +1,13 @@
+# System.Runtime.InteropServices.JavaScript
+
+``` diff
+ namespace System.Runtime.InteropServices.JavaScript {
+     [CLSCompliantAttribute(false)]
+     [EditorBrowsableAttribute(EditorBrowsableState.Never)]
+     [SupportedOSPlatformAttribute("browser")]
+     public sealed class JSFunctionBinding {
+-        public JSFunctionBinding();
```

Comment:
Looking into this

Reply:
Seems to be caught with API Compat and suppressed: https://github.com/dotnet/runtime/blob/451d84e82f4f9c878f530c294d589802eb326da6/src/libraries/apicompat/ApiCompatBaseline.NetCoreAppLatestStable.xml#L6

This was the PR: https://github.com/dotnet/runtime/pull/76196